{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -51,3 +52,9 @@ docs: ## Build docs using Sphinx and copy to docs folder (this makes it easy to
 
 ipython: ## Provides an interactive ipython prompt
 	docker exec -it $(CONTAINER_NAME) ipython
+
+setup-dvc: ## Initializes DVC in project repo
+	docker exec -it $(CONTAINER_NAME) dvc init
+
+setup-dvc-remote: ## Adds DVC remote to repo.
+	docker exec -it $(CONTAINER_NAME) dvc remote add --default default_remote $(DVC_REMOTE_URL)
```

Comment:
Question: Should we put the DVC pull/push commands here? I think you're choosing not to -- just wanted to ask the question explicitly. That makes sense to me -- cause it's like git pull/push -- these are commands you should know and call explicitly and not use the MakeFile for yeah?

Reply:
correct

Reply:
and by that same pattern I agree with you that we should remove `setup-dvc-remote` from Makefile.